### ISSUE TITLE
plugin is not exported in UMD version

### ISSUE THREAD

**Issue by mila76** [probil/vue-socket.io-extended#419]
Seem not working at all

```html
<!doctype html>
<html>
<body>
<div id="q-app">
</div>

<script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/socket.io-client/dist/socket.io.slim.js"></script>
<script src="https://cdn.jsdelivr.net/npm/vue-socket.io-extended"></script>
<script>
  var socket = io('http://url:1923');
  Vue.use(VueSocketIOExt, socket);

  const b = new Vue({
    el: '#q-app',
    mounted() {
      console.log(this.$socket);
     }
  });
</script>
</body>
</html>
```

console undefinied

I have the same without umd working... maybe doc is wrong??
I can't figure out

**Comment by probil** [comment]
Hi @mila76 👋 
I'll check it out. Thanks for your issue!

**Comment by probil** [comment]
@mila76 there is an issue. 

You can apply this quick fix for now.
```diff
- Vue.use(VueSocketIOExt, socket);
+ Vue.use(VueSocketIOExt.default, socket);
```

**Comment by mila76** [comment]
confirm fix work, thanks...
i can continue on my testing :dancer:

**Comment by probil** [comment]
@mila76 Good luck!

**Comment by probil** [comment]
Fixed in `v4.0.1`

Proof:
```html
<!doctype html>
<html>
<body>
<div id="q-app">
</div>

<script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/socket.io-client/dist/socket.io.slim.js"></script>
<script src="https://cdn.jsdelivr.net/npm/vue-socket.io-extended@4.0.1"></script>
<script>
  var socket = io('http://url:1923');
  Vue.use(VueSocketIOExt, socket);

  const b = new Vue({
    el: '#q-app',
    mounted() {
      console.log(this.$socket);
     }
  });
</script>
</body>
</html>
```
